ecpy/utils/widgets/dict_editor.enaml

```diff
@@ -18,10 +18,15 @@ from __future__ import (division, unicode_literals, print_function,
 
 from atom.api import (Atom, Value, Callable, List, Typed, Bool, Subclass,
                       Signal, Dict)
-from enaml.widgets.api import Container
+from collections import OrderedDict
+
+from enaml.widgets.api import (Container, Field, GroupBox)
+from enaml.layout.api import hbox
 
 from .list_editor import ListEditor
 from ..container_change import ContainerChange
+from .qt_completers import QtLineCompleter
+from ...tasks.tools.string_evaluation import EVALUATER_TOOLTIP
```

Comment:
This is a leakage of task into utils please remove it and provide a way to query the tool_tip on the model.